theme/snippets/knowledge-area-markdown.liquid

```diff
@@ -1,3 +1,5 @@
+{% comment %} TO-DO: Have Knowledge Areas use the Markdown snippet. {% endcomment %}
```

Comment:
Will refactor this snippet I made for Knowledge Areas when we work on Knowledge Areas in v2.